pkg/workflows/create.go

```diff
@@ -365,3 +375,50 @@ func (s *DeleteBootstrapClusterTask) Run(ctx context.Context, commandContext *ta
 func (s *DeleteBootstrapClusterTask) Name() string {
 	return "delete-kind-cluster"
 }
+
+func installCuratedPackages(ctx context.Context, spec *cluster.Spec, packagesLocation string) error {
+	err := installPackagesController(ctx, spec)
+	if err != nil {
+		logger.MarkFail("Error when installing curated packages on workload cluster; please install through eksctl anywhere install packagecontroller command", "error", err)
+		return nil
+	}
+
+	err = installPackages(ctx, spec.Cluster.Name, packagesLocation)
+	if err != nil {
+		logger.MarkFail("Error when installing curated packages on workload cluster; please install through eksctl anywhere create packages command", "error", err)
+	}
+	return nil
+}
+
+func installPackagesController(ctx context.Context, spec *cluster.Spec) error {
+	logger.Info("Installing curated packages controller on workload cluster")
+	kubeConfig := kubeconfig.FromClusterName(spec.Cluster.Name)
+	deps, err := curatedpackages.NewDependenciesForPackages(ctx, kubeConfig)
+	if err != nil {
+		return err
+	}
+	chart := spec.VersionsBundle.VersionsBundle.PackageController.HelmChart
+	pc := curatedpackages.NewPackageControllerClient(deps.Helm, deps.Kubectl, kubeConfig, chart.Image(), chart.Name, chart.Tag())
+	err = pc.InstallController(ctx)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func installPackages(ctx context.Context, clusterName, packagesLocation string) error {
+	kubeConfig := kubeconfig.FromClusterName(clusterName)
+	deps, err := curatedpackages.NewDependenciesForPackages(ctx, kubeConfig)
+	if err != nil {
+		return err
+	}
+	packageClient := curatedpackages.NewPackageClient(
+		nil,
+		deps.Kubectl,
+	)
+	err = packageClient.CreatePackages(ctx, packagesLocation, kubeConfig)
```

Comment:
We should already be validating whether the  `packagesLocation` is a dir or not by now. Apart from that, this method contains the logic to ensure the input is validated?

Reply:
This method forwards all that is needed to kubectl. Kubectl will fail during execution in case the path doesn't exist.